satellite/metainfo/metainfo.go

```diff
@@ -124,6 +131,28 @@ func (endpoint *Endpoint) CreateSegment(ctx context.Context, req *pb.SegmentWrit
 		return nil, status.Errorf(codes.Unauthenticated, err.Error())
 	}
 
+	// Check if this projectID has exceeded alpha usage limits, i.e. 25GB of bandwidth or storage used in the past month
+	// TODO: remove this code once we no longer need usage limiting for alpha release
+	// Ref: https://storjlabs.atlassian.net/browse/V3-1274
+	projectID := keyInfo.ProjectID
+	from := time.Now().AddDate(0, 0, -accounting.AvgDaysInMonth) // past 30 days
+	bandwidthTotal, err := endpoint.accountingDB.ProjectBandwidthTotal(ctx, projectID, from)
```

Comment:
Doing these queries for each segment might hit the database quite hard, but not sure whether there's an easy better approach.